### ISSUE TITLE
Buttons in button group overflow menu lack focus indication

### ISSUE THREAD

**Issue by Jinnie** [vmware/clarity#3438]
## Describe the bug

Button group overflow items are focusable and keyboard accessible by tabbing, but lack visual focus indication.

## How to reproduce

1.  Go to https://clarity.design/documentation/button-group#examples
2.  Scroll down to Angular Component / first example
3.  Click on one of the items to set focus to it, then tab to the overflow button.
4.  Open the overflow menu and keep tabbing.
5.  Observe that no focus indication is visible.
<img width="522" alt="Screen Shot 2019-06-11 at 4 13 54 PM" src="https://user-images.githubusercontent.com/1798828/59275094-5740e600-8c64-11e9-848a-67f212b5e6ca.png">

## Expected behavior

A focus outline or other visual indication should appear. It could be a native or native-like outline, or (as done in the dropdown) the same background change as on mouseover.

**Comment by Jinnie** [comment]
This is covered in one of the cases of #3515 
Closing as duplicate in favor of the more detailed issue above.
